authentication/src/main/java/io/camunda/authentication/CamundaOidcUserService.java

```diff
@@ -89,4 +94,11 @@ public OidcUser loadUser(final OidcUserRequest userRequest) throws OAuth2Authent
                 .map(GroupEntity::name)
                 .toList()));
   }
+
+  private String getUsernameFromClaims(final Map<String, Object> claims) {
+    final var usernameClaimName =
```

Comment:
:upside_down_face: I'd just set this once in the constructor instead of keeping the entire `securityConfiguration` around